docs/src/main/asciidoc/deploying-to-openshift-native-howto.adoc

```diff
@@ -0,0 +1,118 @@
+////
+This guide is maintained in the main Quarkus repository
+and pull requests should be submitted there:
+https://github.com/quarkusio/quarkus/tree/main/docs/src/main/asciidoc
+////
+[id="deploying-to-openshift-native-howto"]
+= Deploy {project-name} applications compiled to native executables
+
+include::_attributes.adoc[]
+:diataxis-type: howto
+:categories: cloud, native
+:summary: This guide describes how to deploy a Quarkus application to {openshift} compiled to native executables.
+:topics: devops,kubernetes,openshift,cloud,deployment
+:extensions: io.quarkus:quarkus-openshift
+
+You can deploy your native {project-name} applications to {openshift} compiled to native executables by using the Docker build strategy.
+
+You must create a native executable for your application that targets the Linux AMD64 operating system.
+If your host operating system is different from this, create a native Linux executable by using a container runtime, for example, Docker or Podman.
```

Comment:
_from this_ you mean from Linux AMD64? I think what you want to say is that target OCP architecture must match architecture of the OS where the native executable were built. this needs to be rewritten

Reply:
Updated per your feedback.